test/functional/cmdLineTests/jfr/jfrevents.xml

```diff
@@ -104,4 +104,11 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 		<output type="required" caseSensitive="yes" regex="no">topAddress</output>
 		<output type="failure" caseSensitive="yes" regex="no">jfr print: could not read recording</output>
 	</test>
+	<test id="test jfr SystemGC - approx 30 seconds" platforms="linux.*">
```

Comment:
this isnt specific to linux so it doesnt need `platforms="linux.*"`

Reply:
Fixed

Reply:
you also probably need to trigger a System.gc() in WorkLoad.java, maybe after a thread completes

Reply:
also, I tried a sample application and didnt see the system GC event

Reply:
The test should work now.

> also, I tried a sample application and didnt see the system GC event

Did you call the `System.gc()`? It seems working for me.